.github/workflows/env-setup.yml

```diff
@@ -22,6 +22,8 @@ jobs:
       - name: Download the files and restore the index
         run: |
           aws s3 cp s3://$BUCKET_NAME/$MOST_RECENT_BACKUP_DIR $MOST_RECENT_BACKUP_DIR --recursive
-          export ALGOLIA_INDEX=asap-hub_research_outputs_CI-$(jq --raw-output .pull_request.number "$GITHUB_EVENT_PATH")
           export ALGOLIA_BACKUP=$(readlink -f $MOST_RECENT_BACKUP_DIR)
+          export ALGOLIA_RO_INDEX=asap-hub_research_outputs_CI-$(jq --raw-output .pull_request.number "$GITHUB_EVENT_PATH")
```

Comment:
I know this was existing code, but I think we can get the PR number from the event: github.event.number